docker-compose.yml

```diff
@@ -119,12 +119,12 @@ services:
         +yLvICQmVjHWsNWD3uuT3gEGdeKDIzEnovXUmqhQE7N/YDupTv5hkh8hMKl12XGs
         3WJ4bVSmp3MOQ2EF4zrAl4fXEd4lS3JxKtzbHFbSogAmPFnuP8wIDWTDgfEq
         -----END RSA PRIVATE KEY-----
-      - LAB_RESULTS_MESH_RECIPIENT_MAILBOX_ID_MAPPINGS=000000004400001=lab_results_mailbox
-          000000024600002=lab_results_mailbox
-          000000183500005=lab_results_mailbox
-      - LAB_RESULTS_MESH_POLLING_CYCLE_MINIMUM_INTERVAL_IN_SECONDS=300
-      - LAB_RESULTS_MESH_POLLING_CYCLE_DURATION_IN_SECONDS=285
-      - LAB_RESULTS_MESH_CLIENT_WAKEUP_INTERVAL_IN_MILLISECONDS=60000
+      - LAB_RESULTS_MESH_RECIPIENT_MAILBOX_ID_MAPPINGS=000000004400001=lab_results_ack_mailbox
+          000000024600002=lab_results_ack_mailbox
+          000000183500005=lab_results_ack_mailbox
+      - LAB_RESULTS_MESH_POLLING_CYCLE_MINIMUM_INTERVAL_IN_SECONDS=3
+      - LAB_RESULTS_MESH_POLLING_CYCLE_DURATION_IN_SECONDS=5
+      - LAB_RESULTS_MESH_CLIENT_WAKEUP_INTERVAL_IN_MILLISECONDS=10000
```

Comment:
So we'll scan the MESH mailbox for new messages every 10 seconds (changed from 60 seconds). What's the reason for this change? Is this a new requirement from the business for performance of the adaptor?

Reply:
That's only for the docker-compose test environment to speed up testing.

Reply:
Ah sorry thought this was the app config (and not docker), yeah that's fine